<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>444</width>
    <height>513</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Вход/Регистрация</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x1:0.153682, y1:0.199, x2:1, y2:1, stop:0.0227273 rgba(67, 128, 80, 255), stop:1 rgba(159, 137, 137, 255));</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>140</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 33, 92); font-size: 15px;
background-color: rgb(255, 219, 87);</string>
   </property>
   <property name="text">
    <string>Логин</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>220</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 33, 92); font-size: 15px;
background-color: rgb(255, 155, 157);</string>
   </property>
   <property name="text">
    <string>Пароль</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="username">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>140</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>IBeamCursor</cursorShape>
   </property>
  </widget>
  <widget class="QPushButton" name="signup">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>60</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>РЕГИСТРАЦИЯ</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>60</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(88, 163, 255);</string>
   </property>
   <property name="text">
    <string>ВХОД</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>411</width>
     <height>371</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(76, 246, 255);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="status">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>200</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0); font-size: 15px;
background-color: rgb(255, 110, 166);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="login_btn">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>250</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 155, 217);</string>
    </property>
    <property name="text">
     <string>ВОЙТИ</string>
    </property>
   </widget>
   <widget class="QPushButton" name="show_hide_pass_btn">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>120</y>
      <width>31</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(238, 143, 255);</string>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="password">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>120</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 168, 106);</string>
    </property>
   </widget>
   <zorder>login_btn</zorder>
   <zorder>status</zorder>
   <zorder>show_hide_pass_btn</zorder>
   <zorder>password</zorder>
  </widget>
  <zorder>frame</zorder>
  <zorder>signup</zorder>
  <zorder>label</zorder>
  <zorder>label_3</zorder>
  <zorder>label_2</zorder>
  <zorder>username</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
